Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ExternalDB/TableTesting/TargetTypeSelection.tsx

```diff
@@ -31,7 +31,7 @@ interface IProps {
 }
 
 const parentTableOptions = ['Meter', 'Location', 'Customer', 'Asset',
-    "Line", "Line Segment", "Breaker", "Bus", "Capacitor Bank", "Capacitor Bank Relay", "Transformer", "DER"].map(name => { return { Value: name, Label: name } });
+    "Line", "Line Segment", "Breaker", "Bus", "Capacitor Bank", "Capacitor Bank Relay", "Transformer", "DER", "Generation", "Station Auxilary", "Station Battery"].map(name => { return { Value: name, Label: name } });
```

Comment:
If we use `OpenXDA.List.AssetTypes.concat('Meter', 'Location', 'Customer', 'Asset')` we don't need to modify this any time we have a new Asset Type